systemtest/src/main/java/io/strimzi/systemtest/resources/operator/specific/OlmResource.java

```diff
@@ -72,7 +72,7 @@ public void create() {
         }
 
         // Make sure that operator will be created
-        TestUtils.waitFor("Cluster Operator deployment creation", Constants.GLOBAL_POLL_INTERVAL, CR_CREATION_TIMEOUT,
+        TestUtils.waitFor("ClusterOperator deployment creation", Constants.GLOBAL_POLL_INTERVAL, CR_CREATION_TIMEOUT,
```

Comment:
In `SetupClusterOperator` you have `Cluster Operator`, here it is together

Reply:
Good catch!